src/USER-REACTION/fix_bond_react.cpp

```diff
@@ -575,17 +577,13 @@ FixBondReact::~FixBondReact()
   delete [] set;
 
   if (group) {
-    char **newarg;
-    newarg = new char*[2];
-    newarg[0] = master_group;
-    newarg[1] = (char *) "delete";
-    group->assign(2,newarg);
+    std::string cmd = fmt::format("{} delete",master_group);
```

Comment:
This could be just one line: `group->assign(fmt::format("{} delete",master_group));` and the same for `group->assign(fmt::format("{} delete",exclude_group));`
An alternate way to do the same - and perhaps more readable since it retains the order of arguments - would be:
`group->assign(std::string(master_group) + " delete");` and `group->assign(std::string(exclude_group) + " delete");`

Reply:
thanks!